models/deepseek_v4_flash_dspark/decode_sparse_attn_csa.py

```
@@ -667,9 +989,50 @@ def init_window_block_table():
+    def init_csa_pages():
+        pages = []
+        for request in range(batch):
+            physical = list(
+                reversed(
+                    range(
+                        request * csa_pages_per_request,
+                        (request + 1) * csa_pages_per_request,
+                    )
+                )
+            )
+            for local, page in enumerate(physical):
```

Ruff (0.16.1)

[warning] Loop control variable `local` not used within loop body

Rename unused `local` to `_local`

(B007)